src/MultiReport/Keyboard.cpp

```diff
@@ -98,6 +98,12 @@ void Keyboard_::end(void) {
 }
 
 int Keyboard_::sendReportUnchecked(void) {
+  Serial.print(F("sending nkro report:"));
+  for (byte i{0}; i < 29; ++i) {
+    Serial.print(F(" "));
+    Serial.print(keyReport.allkeys[i], HEX);
+  }
+  Serial.println();
```

Comment:
This is all just debug output, making it clear when the keyboard is in report protocol mode.